src/webgl/p5.Shader.js

```diff
@@ -579,9 +579,6 @@ p5.Shader = class Shader {
     const modelViewProjectionMatrix = modelViewMatrix.copy();
     modelViewProjectionMatrix.mult(projectionMatrix);
 
-    if (this.isStrokeShader()) {
-      this.setUniform('uPerspective', this._renderer._curCamera.useLinePerspective ? 1 : 0);
```

Comment:
I think we use `uPerspective` uniform in used in our `line.vert` file here:
> https://github.com/processing/p5.js/blob/4fbbaf514856ec11b771f701dab5dea7e7f14267/src/webgl/shaders/line.vert#L31

So, we could keep them without the if block as well. Otherwise we would not be able to use `uPerspective` unfiorm and could throw errors.